### DISCUSSION TITLE
Is there a dialect for MongoDB?

### DISCUSSION THREAD

**Discussion by wiseaidev** [sqlalchemy/sqlalchemy#8872] [Usage Questions]
Hey everyone. I am trying to connect a FastAPI app to MongoDB using SQLAchemy. I am wondering if SQLAchemy has a dialect for this; I tried `mongodb`, but it didn't work. So, my guess is that SQLAchemy doesn't support it yet, unfortunately.

**Comment by CaselIT** [comment]
Hi,

mongodb is not a relational db and it doesn't as far as I know support sql, so supporting it would not be easy. there seem to a mongodb dialect for sqlalchemy but I can't say how functional it is https://www.cdata.com/drivers/mongodb/python/
